pg_search/src/api/operator.rs

```diff
@@ -206,42 +199,39 @@ unsafe fn make_search_config_opexpr_node(
     (*var).vartypmod = att.atttypmod;
     (*var).varcollid = att.attcollation;
 
-    let have_query = query.is_some();
-    if let Some(query) = query {
-        // fabricate a `SearchConfig` from the above relation and query string
-        // and get it serialized into a JSONB Datum
-        let search_config = SearchConfig::from((query, &indexrel));
-
-        let search_config_json =
-            serde_json::to_value(&search_config).expect("SearchConfig should serialize to json");
-        let jsonb_datum = JsonB(search_config_json).into_datum().unwrap();
+    // we're about to fabricate a new pg_sys::OpExpr node to return
+    // that represents the `@@@(anyelement, jsonb)` operator
```

Comment:
it's not `@@@(anyelement, jsonb)` anymore -- it should be `@@@(anyelement, paradedb.searchqueryinput)`